config/locales/en.yml

```diff
@@ -1364,6 +1364,7 @@ en:
       in_progress_at_wca_live_html: "This competition is ongoing. You can check the live results <a href='%{link_here}'>here</a>!"
       reg_opens_too_early: "Registration is set to start in less than the required 48 hours."
       registration_already_closed: "The deadline to register has already passed. Please set it to be in the future."
+      uses_v2_reg: "This competition is using the new Registration System. Please contact WCAT if you believe this competition should use the old system instead."
```

Comment:
Suuuuuper nitpick, but could you please not use `reg` in this key? (Properly name it `registration` instead)

Reply:
Fixed